python-package/xgboost/core.py

```diff
@@ -610,9 +611,9 @@ def _init_from_dt(self, data, nthread):
                 ptr = col.data_pointer
                 ptrs[icol] = ctypes.c_void_p(ptr)
         else:
-            # datatable<=0.8.0
-            from datatable.internal import frame_column_data_r  # pylint: disable=no-name-in-module,import-error
+            import datatable.internal.frame_column_data_r # pylint: disable=unsubscriptable-object,no-name-in-module,import-error,unused-import,import-outside-toplevel
             for icol in range(data.ncols):
+                # pylint: disable=undefined-variable
```

Comment:
why we need to add all these stuffs in python package?

Reply:
python lint is failing caused by other prs...